src/convertToObject.js

```diff
@@ -7,6 +7,18 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const result = {};
+  const strings = sourceString.split(';');
```

Comment:
Directly splitting the sourceString by ';' will include empty strings if there are consecutive semicolons or semicolons at the end of the string. Consider using a more robust approach to handle this, such as filtering out empty strings or trimming the input string before splitting.